src/main/java/IslandBFS.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+
+public class IslandBFS {
+
+    int[][] multi;
+    Queue<Point> queue = new LinkedList<Point>();
```

Comment:
This queue's scope shouldnt be this. You will need this only when expanding from a land, so this can be declared in a function where you do the bfs.